music-app/src/components/Helloworld.js

```diff
@@ -1,13 +1,24 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
+import { Mixpanel } from '../analytics/Mixpanel';
 
 export class Helloworld extends Component {
-    render() {
-        return (
-            <div>
-                <h1>Hello World</h1>
-            </div>
-        )
-    }
+  //   componentDidMount() {
+  //     // Mixpanel Tracking
+  //     Mixpanel.identify(this.props.user.display_name);
+  //     Mixpanel.track('Successful Logout');
+  //     Mixpanel.people.set({
+  //       $first_name: this.props.user.first_name,
+  //       $last_name: this.props.user.last_name,
+  //     });
+  //   }
+
+  render() {
+    return (
+      <div>
+        <h1>Hello World</h1>
+      </div>
+    );
+  }
 }
 
-export default Helloworld
+export default Helloworld;
```

Comment:
Why is our component called Helloworld? Seems not like a release-able naming  :D